db/migrate/20180926152238_fix_default_tenant_group.rb

```diff
@@ -36,8 +36,6 @@ def self.create_tenant_group(tenant)
       create_with(
         :description      => "Tenant #{tenant.name} #{tenant.id} access",
         :sequence         => 1,
-        :guid             => SecureRandom.uuid,
-        :miq_user_role_id => role.try(:id)
```

Comment:
miq_user_role_id and guid were no longer in MiqGroup at this time in the migrations, see their commit